### PR TITLE
Modify definition list formatting and wording

### PR DESCRIPTION
**Pull request by andychase**

### CODE REVIEW

File: bip-0099.mediawiki [hunk 1]
```diff
@@ -25,19 +25,17 @@ deployment of changes.
...
-  this kind of changes and they also have some deployment advantages.
-- Hardfork: an intentional consensus fork that is not a softfork.
+;Software fork
+: A copy of an existing project where management is separate from the existing project.
+
+;Consensus fork
+: A divergence in the implementation of the verification rules can impede the expected eventual convergence of the network in a single chain that has the most proof of work and also satisfies the rules. This can be intentional or be caused by a bug in consensus validation reimplementations.
+
+;Softfork
+: A change to the Bitcoin protocol wherein valid blocks/transactions become invalid. Requires a majority of the miners to upgrade to enforce the new rules.
```
Comment (instagibbs) [inline bip-0099.mediawiki]: I think we need to be clear that the majority of hashrate isn't _needed_ to soft-fork(full nodes can do it themselves as long as someone is hashing on their chain), but but miners can _force_ others to a new ruleset.

File: bip-0099.mediawiki [hunk 2]
```diff
@@ -25,19 +25,17 @@ deployment of changes.
...
+
+;Hardfork
+: A change to the Bitcoin protocol that makes invalid blocks valid. Requires all users to upgrade.
```
Comment (instagibbs) [inline bip-0099.mediawiki]: Maybe add "previously" before invalid?

File: bip-0099.mediawiki (hunk 1 above)
Comment (andychase) [inline bip-0099.mediawiki]: I added these changes. I don't think the detail about nodes forking should be added here, nodes can fork at any time by new rule whether it's a soft fork or hard fork. But you are right what's special about soft-forks is that they are backwards compatible.

File: bip-0099.mediawiki [hunk 3]
```diff
@@ -25,19 +25,17 @@ deployment of changes.
...
-- Softfork: an intentional consensus fork where everything that
-  was previously invalid remains invalid, but some things that were
-  valid also become invalid. Majority of miners can impose
-  this kind of changes and they also have some deployment advantages.
-- Hardfork: an intentional consensus fork that is not a softfork.
+;Software fork
+: A copy of an existing project where management is separate from the existing project.
+
+;Consensus fork
+: A divergence in the implementation of the verification rules can impede the expected eventual convergence of the network in a single chain that has the most proof of work and also satisfies the rules. This can be intentional or be caused by a bug in consensus validation reimplementations.
```
Comment (jtimon) [inline bip-0099.mediawiki:32]: is there a reason to use long single lines? It makes it harder to correct things in parallel on the same text (not a huge advantage for multiple lines, but, still, an advantage).

Comment (andychase) [inline bip-0099.mediawiki:32]: Yeah in the syntax for definitions (See [mediawiki: syntax](https://www.mediawiki.org/wiki/Help:Formatting) you have to use one long line for definitions). I tried to wrap the lines originally.

Comment (jtimon) [inline bip-0099.mediawiki:32]: Fair enough, needs rebase though.

**Comment by andychase** [comment]
Rebased

**Pull request merged**